### ISSUE TITLE
[BUG] Some Burp elements use the previous theme elements when updating.

### ISSUE THREAD

**Issue by CoreyD97** [CoreyD97/BurpCustomizer#4]
**Describe the bug**
Some Burp elements, such as toggling advanced scope control, use the previous theme's look and feel when being updated. 

**To Reproduce**
1. Set a theme in Burp Customizer
2. Go to the "Scope" tab.
3. Toggle "Use advanced scope control"
4. See that the panel uses the incorrect theme.

**Expected behavior**
The UI elements should all use the same look and feel when being created / modified.

**Screenshots**
![image](https://user-images.githubusercontent.com/1339555/103747670-e954d000-4ffa-11eb-92b5-3e46fce61d14.png)

**Versions (please complete the following information):**
 - OS: All
 - Burp Suite Pro / Community: Both
 - Burp Suite Version [e.g. 2020.12]: 2020.12

**Comment by CoreyD97** [comment]
My attempted fix didn't actually work.

After a little bit of prodding, I've found that this is actually caused by Burp creating the two scope control UI elements when initially loading Burp, but only adding one to the UI, and switching between them, as opposed to hiding the unused one, when toggling the scope control. As such, applying the theme only applies it to the UI elements which are actually loaded at the time. 

I've also been able to reproduce this with Vanilla Burp, confirming this is a Burp issue. I'll raise it with them, and hopefully it'll be fixed in a future release.
